src/generator/AutoRest.Go.Tests/src/tests/glide.lock

```diff
@@ -0,0 +1,19 @@
+hash: e33d98082bdab9f30032ec6f8f9e2539e549ccbe3dfaf45f1a16702217018912
+updated: 2016-08-10T17:03:36.0631005-07:00
+imports:
+- name: github.com/Azure/go-autorest
```

Comment:
Do we need to check-in the lock file or it should be ignored and regenerated everytime?